src/main/resources/templates/info.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="layout">
+<head>
+    <title>Bus Information</title>
+</head>
+<body>
+<div layout:fragment="content" class="container body-content">
+    <h2>Bus Information</h2>
+    <div>
+        <p>You entered postcode <span th:text="${busInfo.postcode}"></span></p>
+        <p th:each="line: ${busInfo.linesOfBusETA}"> <span th:text="${line}"></span></p>

```

Comment:
And then this line of template will need to be updated to pick the ETA text out of each bus that gets looped over.